client/components/shipping/packages/add-package.js

```diff
@@ -79,7 +87,6 @@ const useDefaultField = ( value, updatePackagesField ) => {
 		is_letter: 'envelope' === value,
 		name: null,
 		is_user_defined: true,
-		outer_dimensions: null,
```

Comment:
Side note: I did look into making the server more flexible, but since we're using Enjoi to generate the Joi validation, there's no good way to allow it in JSON schema.